android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/CoinAddressDerivationTests.kt

```diff
@@ -42,10 +42,13 @@ class CoinAddressDerivationTests {
         CALLISTO -> assertEquals("0x3E6FFC80745E6669135a76F4A7ce6BCF02436e04", address)
         DASH -> assertEquals("XqHiz8EXYbTAtBEYs4pWTHh7ipEDQcNQeT", address)
         DIGIBYTE -> assertEquals("dgb1qtjgmerfqwdffyf8ghcrkgy52cghsqptynmyswu", address)
+
```

Comment:
This expected?

Reply:
Yep, I added this to just make it a bit more readable. We have exactly the same in Kotlin Java tests:
https://github.com/trustwallet/wallet-core/blob/374da804d4a7c7d868ad2ab66d931b785c3d8e37/kotlin/wallet-core-kotlin/src/commonTest/kotlin/com/trustwallet/core/test/CoinAddressDerivationTests.kt#L37-L45